dashboard/lib/dynamic_config/adapters/dynamodb_adapter.rb

```diff
@@ -0,0 +1,64 @@
+# A DynamoDB adapter class that allows us to use
+# dynamodb as a persistent store with the datastore_cache
```

Comment:
Miniscule nit:  Our style guide says to use punctuation in comments when writing sentences.